pmm-app/src/pmm-qan/panel/components/Overview/components/QanTable/Table.tsx

```diff
@@ -127,77 +128,72 @@ export const Table: FC<TableProps> = ({
     onSortChange((state as any).sortBy);
   }, [(state as any).sortBy]);
 
+  const RenderHeader = (headerGroup: HeaderGroup) => (
+    <div {...headerGroup.getHeaderGroupProps()} className={cx('tr', styles.headerRow)}>
+      {headerGroup.headers.map((column: any) => {
```

Comment:
Why is this type `any`?

Reply:
I'm not sure. This PR was to add the search feature to QAN table and not an attempt to refactor it. I just moved this code because it was needed for the feature. I didn't do almost any refactor here because that would take a lot more effort. This is valid for the other comments bellow. But I can see that you understand by those comments that some parts are messy and would require much more time for a proper refactor.

Reply:
I added `HeaderGroup` but didn't have time to type the rest :)